lib/less/core.less

```diff
@@ -211,15 +211,34 @@ ul.rw-list {
 }
 
 ul.rw-list {
-
   &.rw-list-grouped {
-
-    > li.rw-list-optgroup {
-      padding-left: 10px;
-    }
-
-    > li.rw-list-option {
-      padding-left: 20px;
+    > li {
+      &.rw-list-depth {
+        // TODO: There's probably some kind of expression syntax for this...
```

Comment:
I think you're looking for the `@for` control directive here

```
@for $i from 0 through 5
  &-#{$i}
    width: ($i * 10) + 10px
```

or something like that.